pages/governmentresumeguide.md

```diff
@@ -0,0 +1,75 @@
+#Government-style Resume Guide
+
+**Full legal name (First Middle Last)** <br>
```

Comment:
Let's add "Preferred name" under "Full legal name" to make sure we refer to folks the name they prefer.